### PR TITLE
bump targetSdkVersion

### PR DESCRIPTION
**Pull request by KazuCocoa**
Depends on _Support script for locale #23_
I'll rebase or merge this after #23.
After that, I'll request review.

----

The main purpose in this PR is building an apk with _targetSdkVersion 28_

- https://github.com/appium/io.appium.settings/commit/0cd3bf34443f01049f8c6a3978eef3f94a46cb8a
    - API Level 26+ requires https://developer.android.com/about/versions/oreo/background 
    - Changed to register `intent-filter`s dynamically
- https://github.com/appium/io.appium.settings/commit/7bad31457bd6957768da9aa2fee9fcacad3126ce
    - Without `android:exported="true"`, below error happened when I call `adb -s emulator-5556 shell am broadcast -a io.appium.settings.location -n io.appium.settings/.receivers.LocationInfoReceiver`
        ```
        2018-10-27 21:17:46.834 1685-1702/system_process W/BroadcastQueue: Permission Denial: broadcasting Intent { act=io.appium.settings.location flg=0x400010 cmp=io.appium.settings/.receivers.LocationInfoReceiver } from null (pid=19590, uid=2000) is not exported from uid 10087 due to receiver io.appium.settings/.receivers.LocationInfoReceiver
        ```
    - After adding it,
        ```
        $ adb -s emulator-5556 shell am broadcast -a io.appium.settings.location -n io.appium.settings/.receivers.LocationInfoReceiver
        Broadcasting: Intent { act=io.appium.settings.location cmp=io.appium.settings/.receivers.LocationInfoReceiver }
        Broadcast completed: result=-1, data="30.00000 -72.00000 3.00000"
        ```

I tested all of commands by manual with Android 8.1 and 7.1. They worked fine.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -81,34 +84,12 @@
...
-                <action android:name="io.appium.settings.locale" />
-            </intent-filter>
-        </receiver>
-
-        <receiver android:name=".receivers.LocationInfoReceiver">
-            <intent-filter>
-                <action android:name="io.appium.settings.location" />
-            </intent-filter>
-        </receiver>
+        <receiver android:name=".receivers.WiFiConnectionSettingReceiver" />
```
Comment (mykola-mokhnach) [inline app/src/main/AndroidManifest.xml]: actually all of them can be defined as exported

**Comment by KazuCocoa** [comment]
Yes. Will update this branch soon.

**Comment by KazuCocoa** [comment]
📝 I'll test again later and merge.

**Comment by KazuCocoa** [comment]
Tested with OS 6.0, 7.1, 8.1. All of them worked.

**Pull request merged**